src/System.Net.Security/src/System/Net/SSPIInterfaceNego.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.ComponentModel;
+using System.Globalization;
+using System.Net.Security;
+using System.Runtime.InteropServices;
+using System.Security.Authentication.ExtendedProtection;
+using System.Security.Cryptography.X509Certificates;
+using System.Security.Authentication;
+
+namespace System.Net
+{
+    // Used to define the interface for security to use.
+    internal interface SSPIInterfaceNego
+    {
+        void VerifyPackageInfo();
+        Exception GetException(SecurityStatus status);
```

Comment:
See above: `GetException()` could be replaced by throw new `Win32Exception(...)`.